seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -4802,7 +4802,7 @@
         "var ourArr = [ 9, 10, 11, 12];\nvar ourTotal = 0;\n\nfor (var i = 0; i < ourArr.length; i++) {\n  ourTotal += ourArr[i];\n}\n\nvar myArr = [ 2, 3, 4, 5, 6];\nvar total = 0;\n\nfor (var i = 0; i < myArr.length; i++) {\n  total += myArr[i];\n}"
       ],
       "tests": [
-        "assert(code.match(/var\\s*total\\s*=\\s*0\\s*;/), 'message: <code>total</code> should be declared and initialized to 0');",
+        "assert(code.match(/var.*?total\\s*=\\s*0\\s*;/), 'message: <code>total</code> should be declared and initialized to 0');",
```

Comment:
Yes, it will. But the editor should throw an error from not using a `for` properly. Is that what you were referring to?